### DISCUSSION TITLE
Welcome to IoT-For-Beginners Discussions!

### DISCUSSION THREAD

**Discussion by jimbobbennett** [microsoft/IoT-For-Beginners#219] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by princeanire** [comment]
Hello! I'm Andrew, I'm an aspiring software and web developer currently learning the basics of web development, I recently found this repository, and I'm excited to learn more about IoT!

**Comment by TheCSGuy25** [comment]
Hey, Y'all can call me Panda/Pandz . I'm a recent HS graduate. I am interested in IoT as well as ML. I'm here as though I cannot provide any technical knowledge as I'm not armed with that level of knowledge so I will try to contribute through documentations as much as I can . I hope to meet more nice people who're a part of this Repo !

**Comment by syedaffan01** [comment]
Hello, I'm Rafi and I am currently new to leaning about IoT and related stuff and just found this repo and am very excited about learning through this!

**Comment by Thamara0perera** [comment]
I can't access to videos l, It says videos are private.

**Comment by JordyMail** [comment]
Hi everyone! You can call me Jordy. I’m passionate about IoT and Data Science. I’ve been learning through bootcamps and hands-on projects, and I’m always excited to explore more. While I'm still growing my technical skills, I enjoy contributing through coding, documentation, and teamwork. Looking forward to learning and collaborating with awesome people in this repo!🐦‍🔥
